yarn-project/end-to-end/src/e2e_blacklist_token_contract/access_control.test.ts

```diff
@@ -73,14 +73,14 @@ describe('e2e_blacklist_token_contract access control', () => {
       const newRole = new Role().withAdmin().withAdmin();
       await expect(
         t.asset.withWallet(t.other).methods.update_roles(AztecAddress.random(), newRole.toNoirStruct()).prove(),
-      ).rejects.toThrow("Assertion failed: caller is not admin 'caller_roles.is_admin'");
+      ).rejects.toThrow(/Assertion failed: caller is not admin .*/);
```

Comment:
These changes are due to this bug: https://github.com/noir-lang/noir/issues/5523

It's the only difference in behavior (besides macro fuckery) we've found